data-manager/src/test/groovy/org/orbisgis/datamanager/GroovyH2GISTest.groovy

```diff
@@ -564,6 +564,7 @@ class GroovyH2GISTest {
         h2GIS.getSpatialTable("orbisgis").the_geom.createSpatialIndex()
         assertTrue h2GIS.getSpatialTable("orbisgis").the_geom.indexed
         assertTrue h2GIS.getSpatialTable("orbisgis").the_geom.spatialIndexed
+        h2GIS.getSpatialTable("orbisgis").the_geom.createSpatialIndex()
```

Comment:
This line seems to be useless as there is no assertion after.

Reply:
yes. I will remove it.